elections/2021/SC/README.md

```diff
@@ -0,0 +1,163 @@
+---
+title: "Knative Steering Committee Election - 2021 Voters Guide"
+linkTitle: "2021 Voters Guide"
+weight: 30
+type: "docs"
+---
+
+# 2021 Voters Guide - Knative Steering Committee Election
+
+## Purpose
+
+The role of this election is to fill out the three (3) seats due for election
+this year on the [Knative Steering Committee]. Each elected member will serve a
+two (2) year term.
+
+## Background
+
+This election will shape the future of Knative as a community and project. The
+Knative Steering Committee (SC) is responsible for the general health of the
+Knative community. Seats up for the election are the remaining bootstrap seats
+and in addition to contributor seats, we're introducing one (1) end-user seat to
+bring more diversity into the SC representation.
+
+## Eligibility
+
+Please refer to the [SC Election Charter] for:
+
+- [Eligibility for candidacy]
+- [Eligibility for voting]
+
+The list of eligible voters is stored (TODO: vaikas add this)[in the community repository](./voters.yaml), but it is easier
+to check if you are eligible by logging into [Elekto]. See Voting Process below for more details.
+
+## Schedule
+
+| Date         | Event                    |
+| ------------ | ------------------------ |
+| October 25   | Announcement of Election, call for nominations |
+| November 8   | All candidate nominations and voting exception forms due by 0000 UTC (5pm Pacific) |
```

Comment:
Since the exception forms are handled in Elekto, we should change this to:
| November 8   | All candidate nominations due by 0000 UTC (5pm Pacific)

and add this one:
| November 19 | Voter exception requests due by 0000 UTC (5pm Pacific)

I pulled the date in a bit so that we could handle the exceptions in the week before the US Thanksgiving holiday.

Reply:
Done 👍